package.json

```diff
@@ -1,29 +1,138 @@
 {
     "name": "metadata-skeleton-offline-2",
```

Comment:
Why metadata-skeleton-offline-2?
Its referenced in the README too, so i guess its a leftover from development.

Also, speaking of README, can we update it and add instructions of how to launch the Metadata JSON Merge tool?